platforms/android/run_module_app/README.md

```diff
@@ -0,0 +1,47 @@
+# iree-run-module Android App
+
+This directory contains an Android application packing the `iree-run-module`
+command-line tool together with a specific IREE VM module invocation.
+This is helpful for profiling/debugging using Android tools.
+
+## Native Activity
+
+The App uses Android [`NativeActivity`][native-activity] to bridge IREE core
+libraries together with the Android system. Native activity allows one to
+implement an Android Activity purely in C/C++. There are
+[tutorials][native-activity-tutorial] and [examples][native-activity-example]
+one can follow to learn about Native Activity.
+
+## Build the APK
+
+We need to package IREE native libraries and a specific VM module invocation.
+IDE is not very convenient to perform such tasks, so a script,
+[`build_apk.sh`](./build_apk.sh), is provided to automate the process.
```

Comment:
We'll need a better story for going from Python model code to an Android application than this in the future. This might be enough for our own testing, but we'll need simpler, well supported ways that integrate with other project's existing build systems to include IREE code and resources.

@not-jenni has the Java bindings building with an IREE bytecode module through Bazel internally and none of this dance was needed there (though to be fair, the internal Bazel does a lot of work under the covers when building for Android).

Reply:
Yeah. This is just for profiling and developer usage. I've updated the README to make it clear.